README.md

```diff
@@ -90,8 +90,11 @@ as [Wikimedia Germany](https://wikimedia.de) employee for the [Wikidata project]
 
 ### 5.0.0 (dev)
 
-* Updated minimal required PHP version from 5.5.9 to 7.0.
-  In particular, HHVM is no longer supported.
+### 4.1.0 (TBD)
+
+* Type hinted the `$previous` parameter as `Throwable` instead of `Exception`
+* Updated minimum required PHP version from `5.5.9` to `7.2` (HHVM is no longer supported)
```

Comment:
This says that we support PHP from `5.5.9` to `7.2`... while it actually looks like we now support `7.2` to `8.0` (at least that's whats tested on travis)?

Reply:
It says we dropped `5.5.9` as the minimum supported version and that `7.2` is now the minimum supported version. Although, maybe this sentence isn't phrased very well. Feel free to suggest an alternative. :)

Reply:
Huh, seems I simply overlooked the "minimum"... looking at it again, this seems all good.